.rubocop.yml

```diff
@@ -29,6 +29,9 @@ Style/Documentation:
 Lint/UnneededDisable:
   Enabled: false
 
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
This wasn't an issue before 😕 If you call it with `bundle exec` there shouldn't be any problem no ?